server/src/main/java/org/elasticsearch/action/admin/indices/stats/CommonStatsFlags.java

```diff
@@ -217,7 +217,8 @@ public CommonStatsFlags clone() {
         // 14 was previously used for Suggest
         RequestCache("request_cache", 15),
         Recovery("recovery", 16),
-        Bulk("bulk", 17);
+        Bulk("bulk", 17),
+        Shards("shards", 9);
```

Comment:
Should this be 18?

Reply:
Yep. Fortunately an IT test had also found this.